test/gurka/test_truck.cc

```diff
@@ -314,6 +314,46 @@ TEST(Standalone, TruckSpeeds) {
   EXPECT_EQ(de_both_2->truck_speed(), 15);
 }
 
+TEST(StandAlone, TestTruck) {
+  const gurka::ways ways = {
+      {"AB", {{"highway", "secondary"}}},
+      {"BC", {{"highway", "secondary"}}},
+      {"CD", {{"highway", "secondary"}}},
+      {"AE",
+       {
+           {"highway", "secondary"},
+           {"maxheight", "1"},
+           {"hgv:conditional", "yes @ (09:00-18:00)"},
+       }},
```

Comment:
this currently fails on master because `maxheight` gets the access mask `kAuto | kTruck | kTaxi | ...` while `hgv:conditional` only has `kTruck`, so it returns true before maxheight is evaluated.